tox.ini

```diff
@@ -471,7 +474,9 @@ deps =
     -r{toxinidir}/requirements.txt
     standalone-{cpu,gpu,tpu,local}-py{36,37,38,39,310}: -r{toxinidir}/requirements_dev.txt
     pytest-mock
-    yea-wandb=={env:YEA_WANDB_VERSION}
+    https://github.com/wandb/yea/archive/yea-0.8.zip
+    https://github.com/wandb/yea-wandb/archive/yea-wandb-0.8.zip
+    ; yea-wandb=={env:YEA_WANDB_VERSION}
```

Comment:
same as above

Reply:
yea-wandb and yea were released so i have updated